### PR TITLE
Initial commit of page 1

### PR DESCRIPTION
**Pull request by disha2300**
Bulletin board page, containg html and css pages.

### CODE REVIEW

File: public/page1/indexpg.html [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+<script defer src="https://use.fontawesome.com/releases/v5.0.7/js/all.js"></script>
+    <!-- Bootstrap CSS -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+    <link rel="stylesheet" href="staticpg.css" > </head><title>Hello</title>
+  </head>
+  <body>
+
+    <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"> </script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js" integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q" crossorigin="anonymous"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js" integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl" crossorigin="anonymous"></script>
```
Comment (Omkaragrawal) [inline public/page1/indexpg.html:20]: move it to the bottom inside the body tag.

File: public/page1/indexpg.html [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+<link href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,900;1,900&family=Roboto:wght@500&family=Ubuntu:wght@300&display=swap" rel="stylesheet">
+    <!--fontawesome-->
+<script defer src="https://use.fontawesome.com/releases/v5.0.7/js/all.js"></script>
+    <!-- Bootstrap CSS -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+    <link rel="stylesheet" href="staticpg.css" > </head><title>Hello</title>
+  </head>
+  <body>
+
+    <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"> </script>
```
Comment (Omkaragrawal) [inline public/page1/indexpg.html:18]: move it to the bottom inside the body tag.

File: public/page1/indexpg.html [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <!-- Google Fonts -->
+
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+<link href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,900;1,900&family=Roboto:wght@500&family=Ubuntu:wght@300&display=swap" rel="stylesheet">
+    <!--fontawesome-->
+<script defer src="https://use.fontawesome.com/releases/v5.0.7/js/all.js"></script>
```
Comment (Omkaragrawal) [inline public/page1/indexpg.html:11]: move it to the bottom inside the body tag.

**Review by Omkaragrawal — CHANGES_REQUESTED** [review]
move the tags to the bottom inside the body tag.

**Comment by disha2300** [comment]
> move the tags to the bottom inside the body tag.



> move the tags to the bottom inside the body tag.

**Pull request merged**